language_detector/main.py

```diff
@@ -1,9 +1,53 @@
 """This is the entry point of the program."""
 
-from .languages import LANGUAGES
+try:
+    from .languages import LANGUAGES
+except:
+    from languages import LANGUAGES
 
 
 def detect_language(text, languages=LANGUAGES):
     """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    spanish = []
+    english = []
+    german = []
+    #Grant easy access to the dictionaries
+    for lang in languages:
+        if lang["name"] == "Spanish":
+            spanish = lang["common_words"]
+        elif lang["name"] == "English":
+            english = lang["common_words"]
+        elif lang["name"] == "German":
+            german = lang["common_words"]
+            
+    #Create a variable with number of occurrences per language
+    english_occurrences = 0
+    german_occurrences = 0
+    spanish_occurrences = 0
+    
+    #Tokenize the text input
+    tokenized_text = text.split(" ")
+
+    
+    #count occurrences
+    #So we could iterate through that list
+    for word in tokenized_text:
+        #If the word is contained in the english common words list
+        if word in english:
+            #Increase the english occurrences
+            english_occurrences += 1
+            
+        elif word in spanish:
```

Comment:
What if the word appears in more than one list?  if/elif will favor whichever language that comes first in the list.